src/Likvido.ApplicationInsights.Telemetry/ExecuteAsRequest/TelemetryClientExtensions.cs

```diff
@@ -0,0 +1,178 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.ApplicationInsights;
+using Microsoft.ApplicationInsights.DataContracts;
+
+namespace Likvido.ApplicationInsights.Telemetry
+{
+    public static class TelemetryClientExtensions
+    {
+        /// <summary>
+        /// Tracks operation as a request metric
+        /// </summary>
+        /// <param name="client"></param>
+        /// <param name="requestOptions"></param>
+        public static void ExecuteAsRequest(
```

Comment:
Is it possible to make the synchronous version call the asynchronous version and block - instead of implementing the same thing twice?